docs/notebooks/mister_p.ipynb

```diff
@@ -0,0 +1,8615 @@
+{
```

Comment:
<p>others may have the same question as me: where does the "0.25" come from? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bambinos/bambi/pull/716/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<p>Separated out the calculation of mean outcome from weighted average calc to make this clearer</p>